src/aap_eda/core/utils/credentials.py

```diff
@@ -470,3 +478,30 @@ def check_reserved_keys_in_extra_vars(data: dict[str, any]) -> None:
                     "reserved": ", ".join(sorted((RESERVED_EXTRA_VAR_KEYS))),
                 }
             )
+
+
+def build_copy_post_data(eda_credential: "models.EdaCredential") -> dict:
```

Comment:
Why do you use string, instead of class object as the type hint? Seems like the latter is more recommended

Reply:
This is a smart way of avoiding circular import error (which happens when typing is used without quotation marks) that @jshimkus-rh came up with, IIRC.

Reply:
Credit where credit is due. 🫡 @Alex-Izquierdo is the one who told me about it.

Reply:
I used it to avoid rearranging code for the simple inclusion of a new parameter to an existing method.  At some point we may want to take that on, but at least it's still confined to the one file.